packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -280,4 +290,59 @@ export = {
 			return res.json(sharedWorkflow.workflow);
 		},
 	],
+	getWorkflowTags: [
+		authorize(['global:owner', 'global:admin', 'global:member']),
+		async (req: WorkflowRequest.GetTags, res: express.Response): Promise<express.Response> => {
+			const { id } = req.params;
+
+			if (config.getEnv('workflowTagsDisabled')) {
+				return res.status(400).json({ message: 'Workflow Tags Disabled' });
+			}
```

Comment:
Should this be a middleware applied here and also to all tags endpoints in the Public API?

Reply:
Could be an improvement but not in the scope of this PR, as other methods in this same handler won't interact with tags